### ISSUE TITLE
xcode9 + macOS 10.13  Compile error 

### ISSUE THREAD

**Issue by mengzhihoing** [braintree/braintree_ios#359]
### General information

* SDK/Library version: 
* Environment: 
* iOS Version and Device: 
* Integration type and version: 

### Issue description

hi,when i update my xcode and macOS ,it gone wrong,

cocoapods:

# Includes Cards and PayPal
  pod 'Braintree', '~> 4.9.3'  

# Optionally include additional Pods
  #pod 'Braintree/DataCollector'
  #pod 'Braintree/Venmo'

screen shot: http://wenjian.ml/error.png

thx for your help.

**Comment by bluk** [comment]
Thanks for the report. We are investigating and will issue a fix for this.

**Comment by bluk** [comment]
In the meantime, you can try the following suggested by @mdelamata [here](https://github.com/braintree/braintree_ios/issues/353#issuecomment-332873520):

```
post_install do |installer|
      installer.pods_project.targets.each do |target|
            target.build_configurations.each do |config|
                if target.name == "Braintree"
                  config.build_settings['CLANG_WARN_UNGUARDED_AVAILABILITY'] = 'NO'
                end
           end
      end
  end
```

**Comment by mengzhihoing** [comment]
@bluk 
Thank you for your help, according to the above practice, has been successfully compiled.

**Comment by bluk** [comment]
@mengzhihoing This should also be resolved in version [4.9.4](https://github.com/braintree/braintree_ios/releases/tag/4.9.4) released late yesterday without requiring the above fix. If you still have an issue, please leave a comment and we can reopen this issue or open a new issue. Thanks!
